lib/themes/dosomething/paraneue_dosomething/js/onboarding.js

```diff
@@ -17,7 +18,7 @@ $(document).ready(function() {
   const disabledByLanguage = ['mx', 'br'].indexOf(campaign.language.language_code) != -1;
   let localStorageKey = null;
 
-  if (campaign && user && !disabledByLanguage) {
+  if (campaign && user && !disabledByLanguage && !Modal.isOpen()) {
```

Comment:
This conditional is getting long, not sure if we should break it out into a function of its own or just leave it